src/transformStateWithClones.js

```diff
@@ -7,7 +8,45 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
```suggestion
  for (let action of actions) {
```
Цикл for of тут підійшов би більше і було б інтуітивніше